resources/route53_helpers.go

```diff
@@ -0,0 +1,14 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/route53/types"
+)
+
+func getRoute53tagsByResourceID(id string, set []types.ResourceTagSet) []types.Tag {
```

Comment:
move this _types.go or _types.go to _helpers.go no need two files. I prefer the _types.